tfjs-backend-webgpu/src/kernels/strided_slice_webgpu.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {getCoordsDataType} from '../shader_preprocessor';
+import {computeDispatch, flatDispatchLayout} from '../webgpu_util';
+
+import {WebGPUProgram} from './webgpu_program';
+
+export class StridedSliceProgram implements WebGPUProgram {
+  variableNames = ['x'];
+  outputShape: number[];
+  userCode: string;
+  dispatchLayout: {x: number[]};
+  dispatch: [number, number, number];
+  workPerThread = 1;
```

Comment:
Have you tried increasing the workPerThread? If not we can try it in a follow-up PR. Would you mind adding a TODO here to increase the workPerThread?

Reply:
I didn't try it yet, I will try it in a followup PR.And TODO is added.